backend/lib/routes/controllerregistrations.js

```diff
@@ -0,0 +1,31 @@
+//
+// SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+'use strict'
+
+const express = require('express')
+const { controllerregistrations } = require('../services')
+
+const router = module.exports = express.Router()
+
+router.route('/')
+  .get(async (req, res, next) => {
+    try {
+      const user = req.user
+      res.send(await controllerregistrations.list({ user }))
+    } catch (err) {
+      next(err)
+    }
+  })
+
+router.route('/resources/kind/network/types')
```

Comment:
I don't like this pattern since everything in between will return 404.

Reply:
Yes, I thought that you will raise this point. What would you suggest? Do not use a path and hust have an independent route like `/networktypes`?

Reply:
ah just saw your other comment, ok. Let's do it like that